packages/backend/src/services/tsoaServiceContainer.ts

```diff
@@ -28,10 +29,10 @@ const isTsoaControllerCtor = (
  */
 export const iocContainer: IocContainerFactory = () => ({
     async get<T extends RouteControllerKlass>(Ctor: T) {
-        if (isTsoaControllerCtor(Ctor)) {
-            return new Ctor();
+        if (isBaseControllerCtor(Ctor)) {
```

Comment:
Inverted check to ensure we're doing the right thing here, now that this is important (`BaseController` extends `tsoa`'s `Controller`, so the order is important).